toolkit/scripts/pkggen.mk

```diff
@@ -207,20 +210,23 @@ $(preprocessed_file): $(cached_file) $(go-graphPreprocessor)
 pkggen_archive	= $(OUT_DIR)/rpms.tar.gz
 srpms_archive  	= $(OUT_DIR)/srpms.tar.gz
 
-.PHONY: build-packages clean-build-packages hydrate-rpms compress-rpms clean-compress-rpms compress-srpms clean-compress-srpms
+.PHONY: build-packages clean-build-packages hydrate-rpms compress-rpms clean-compress-rpms compress-srpms clean-compress-srpms clean-build-packages-workers
 
 # Execute the package build scheduler.
 build-packages: $(RPMS_DIR)
 
 clean: clean-build-packages clean-compress-rpms clean-compress-srpms
-clean-build-packages:
+
+# Empty rule for clean-build-packages-workers, we will append more rules to this later via the $(eval) function below
+clean-build-packages-workers:
+	@echo Verifying no mountpoints present in $(CHROOT_DIR)
+	$(SCRIPTS_DIR)/safeunmount.sh "$(CHROOT_DIR)"/* && \
```

Comment:
I'm guessing the `CHROOT_DIR` directory isn't ever a mountpoint itself, correct?

Reply:
Its case-by-case. Here each package will have its own unique chroot so the /* makes sense.